raft_test.go

```diff
@@ -1382,6 +1478,23 @@ func snapshotAndRestore(t *testing.T, offset uint64) {
 	// Get the last index before the restore.
 	preIndex := leader.getLastIndex()
 
+	if restoreNewCluster {
+		var c2 *cluster
+		if monotonicLogStore {
+			opts := &MakeClusterOpts{
```

Comment:
We can probably just re-use the cluster opts from above if you scope the opts var to the function. All of the vars are the same.

Reply:
Good call.  Added in https://github.com/hashicorp/raft/pull/545/commits/7d73c137a93834c6d84107c2f14eb332020b809f